pos_ticket_without_price/static/src/js/receiptwithoutprice.js

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2021 Initos Gmbh
+   License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+*/
+odoo.define("point_of_sale.OrderReceiptWithoutPrice", function (require) {
```

Comment:
Please rename this file to `OrderReceiptWithoutPrice.js` and also update it's assets entry